styles/prosilver/template/show_tag.html

```diff
@@ -47,9 +47,11 @@ <h2 class="forum-title">{RH_TOPICTAGS_SEARCH_HEADER}</h2>
 		<!-- EVENT viewforum_body_topic_row_before -->
 		<li class="row<!-- IF topicrow.S_ROW_COUNT is even --> bg1<!-- ELSE --> bg2<!-- ENDIF --><!-- IF topicrow.S_POST_GLOBAL --> global-announce<!-- ENDIF --><!-- IF topicrow.S_POST_ANNOUNCE --> announce<!-- ENDIF --><!-- IF topicrow.S_POST_STICKY --> sticky<!-- ENDIF --><!-- IF topicrow.S_TOPIC_REPORTED --> reported<!-- ENDIF -->">
 			<!-- EVENT viewforum_body_topic_row_prepend -->
-			<dl class="row-item {topicrow.TOPIC_IMG_STYLE}">
```

Comment:
You're right, it appears to be a quirk of the we_universal theme: https://github.com/melvingb/we_universal/blob/master/template/viewforum_body.html#L157

Still, it's one of the most popular ones, pity that it doesn't work out of the box.